api/pkg/handlers/invite/send.go

```diff
@@ -0,0 +1,58 @@
+package invite
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/verasthiago/tickets-generator/api/pkg/builder"
+	"github.com/verasthiago/tickets-generator/api/pkg/validator/invite"
+	"github.com/verasthiago/tickets-generator/shared/auth"
+	"github.com/verasthiago/tickets-generator/shared/errors"
+	"github.com/verasthiago/tickets-generator/shared/models"
+)
+
+type InviteSendAPI interface {
+	Handler(context *gin.Context) error
+}
+
+type InviteSendHandler struct {
+	builder.Builder
+}
+
+func (i *InviteSendHandler) InitFromBuilder(builder builder.Builder) *InviteSendHandler {
+	i.Builder = builder
+	return i
+}
+
+func (i *InviteSendHandler) Handler(context *gin.Context) error {
+	var err error
+	var user *models.User
+	var request *invite.InviteSendRequest
+
+	if err = context.ShouldBindJSON(&request); err != nil {
+		return err
+	}
+
+	if errList := request.ValidateSyntax(); len(errList) > 0 {
+		return errors.CreateGenericErrorFromValidateError(errList)
+	}
+
+	if request.Token, err = auth.GetJWTClaimFromToken(context.GetHeader("Authorization"), i.GetSharedFlags().JwtKey); err != nil {
```

Comment:
Do you have a token on the body req and in the header?? And you cant get the id from the token?

Reply:
Don't have token on body, I just encapsulated the token struct on the request struct so I can access, I can refactor to pass as parameter